.copr/Makefile

```diff
@@ -2,7 +2,49 @@ srpm:
 	dnf -y install git
 	git submodule update --init --recursive
 	cd deployment/fedora-package-x64;                    \
-	./create_tarball.sh;                                 \
+    WORKDIR="$( pwd )"; \
+    VERSION="$( sed -ne '/^Version:/s/.*  *//p' "${WORKDIR}"/pkg-src/jellyfin.spec )"; \
+    package_temporary_dir="${WORKDIR}/pkg-dist-tmp"; \
+    pkg_src_dir="${WORKDIR}/pkg-src"; \
+    GNU_TAR=1; \
+    tar \
+    --transform "s,^\.,jellyfin-${VERSION}," \
+    --exclude='.git*' \
+    --exclude='**/.git' \
+    --exclude='**/.hg' \
+    --exclude='**/.vs' \
+    --exclude='**/.vscode' \
+    --exclude='deployment' \
+    --exclude='**/bin' \
+    --exclude='**/obj' \
+    --exclude='**/.nuget' \
+    --exclude='*.deb' \
+    --exclude='*.rpm' \
+    -czf "${SOURCE_DIR}/SOURCES/pkg-src/jellyfin-${VERSION}.tar.gz" \
```

Comment:
`${SOURCE_DIR}` has no value in the context of COPR simply running `make -f .copr/Makefile srpm` which is how this Makefile works in COPR.